### PR TITLE
Add pydantic to classifiers

### PR DESCRIPTION
**Pull request by yezz123**
Add Pydantic to Classifiers 🚀

### CODE REVIEW

**Comment by samuelcolvin** [comment]
LGTM, thanks.

pydantic is already included in https://pypi.org/classifiers/

**Comment by yezz123** [comment]
> LGTM, thanks.
> 
> pydantic is already included in https://pypi.org/classifiers/

❤️ 

Yes, but Hatch still has not yet been added to the classifiers.

File: pyproject.toml [hunk 1]
```diff
@@ -45,6 +45,8 @@ classifiers = [
     'Environment :: Console',
     'Environment :: MacOS X',
     'Framework :: Hypothesis',
+    'Framework :: Pydantic',
+    'Framework :: Pydantic :: 1',
```
Comment (hramezani) [inline pyproject.toml:49]: I am not sure that we should include a classifier with the version in Pydantic.

I can't see it in [Django](https://github.com/django/django/blob/main/setup.cfg#L13), [FastAPI](https://github.com/tiangolo/fastapi/blob/master/pyproject.toml#L29), and [Flask](https://github.com/pallets/flask/blob/main/pyproject.toml#L11).

Comment (yezz123) [reply pyproject.toml:49]: I see we can delete it 🐛 

```suggestion
```

Comment (samuelcolvin) [reply pyproject.toml]: good catch, agreed.

**Comment by yezz123** [comment]
Hello @samuelcolvin can you re-run the checks, this one is merged in hatch https://github.com/pypa/hatch/pull/725

**Comment by ofek** [comment]
I'll have to release Hatchling, I'll do that over the weekend

**Comment by samuelcolvin** [comment]
@ofek any chance you can update hatchling?

**Comment by ofek** [comment]
https://github.com/pypa/hatch/releases/tag/hatchling-v1.13.0

**Comment by yezz123** [comment]
Thanks @ofek 🙏🏻 !

I guess we can merge it now!

cc @samuelcolvin

**Pull request merged**